docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,983 @@
+# Bold BI auto deployment using kubectl on Kubernetes Service
+
+This section helps you to deploy [Bold BI](https://www.boldbi.com/) in Kubernetes without manually activating licensing and configuring startup from the browser. This also helps us to customize the branding using environment variables.
+
+## Deployment Methods
+
+There are two ways to deploy Bold BI on the Kubernetes cluster. Please refer to the following documents for Bold BI deployment.
+
+* [Deploy Bold BI using kubectl](bold-bi-auto-deployment.md#deploy-bold-bi-using-kubectl)
+* [Deploy Bold BI using Helm](#)
+   
+
+## Deploy Bold BI using kubectl
+
+The below steps helps you do deploy Bold BI using kubectl in a kubernetes cluster.You can create Kubernetes cluster on cloud cluster providers(GKE,AKS and EKS). After completing cluster creation, connect to it and you can download the configuration files [here](../deploy/auto-deployment.zip). This directory includes configuration YAML files, which contains all the configuration settings needed to deploy Bold BI on Kubernetes cluster.
+
+## Deployment prerequisites
+
+The following requirements are necessary to deploy the Bold BI solution using kubectl. 
+
+* [Install Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to deploy Bold BI using kubectl.
+* [File Storage](pre-requisites.md#file-storage)
+* [Create and connect a cluster](pre-requisites.md#create-a-cluster)
+* Load Balancing- [Nginx](https://kubernetes.github.io/ingress-nginx/deploy/)
+
+## Steps for Bold BI auto deployment using kubectl.
+
+The following links explain Bold BI Kubernetes deployment in a specific cloud environments.
+
+
+ * [Azure Kubernetes Service(AKS)](bold-bi-auto-deployment.md#bold-bi-on-microsoft-azure-kubernetes-service)
+ * [Amazon Elastic Kubernetes Service(EKS)](bold-bi-auto-deployment.md#bold-bi-on-amazon-elastic-kubernetes-service)
+ * [Google Kubernetes Engine (GKE)](bold-bi-auto-deployment.md#bold-bi-on-google-kubernetes-engine)
+
+
+### Bold BI on Microsoft Azure Kubernetes Service
+
+1. Download the deployment file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/UMP-22952-Autodeployment-documentation/deploy/auto-deployment/deploy_aks.yaml) to deloy Bold BI on AKS.
+
+2. Navigate to the folder where the deployment files were downloaded from **Step 1**.
+
+3. Create a File share instance in your storage account and note the File share name to store the shared folders for application usage.
```

Comment:
Create a File share instance in your storage account and note the File share name to store the shared folders for application use

Reply:
concern addressed